charts/moco/templates/generated/generated.yaml

```
@@ -372,7 +372,7 @@ spec:
     app.kubernetes.io/component: moco-controller
     app.kubernetes.io/name: '{{ include "moco.name" . }}'
 ---
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: '{{ include "admissionregistration.apiVersion" . }}'
```

single quote can be removed.
```suggestion
apiVersion: {{ include "admissionregistration.apiVersion" . }}
```